CHANGELOG.md

```diff
@@ -6,8 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 
+- [#xxx] Add feature `tfdi` to enable `tfdi` debuggers
```

Comment:
```suggestion
- [#xxx] Add feature `ftdi` to enable `ftdi` debuggers
```

Reply:
Oh wow. Thank you for catching that 😵‍💫

Reply:
Are you using a ftdi debugger and tested it by any chance?

Reply:
Nope. Was just reporting #382 and browsed around. I'm familiar with FTDI chips, so the naming mismatch was pretty visible to me.

Reply:
👍🏾